LMS/library.py

```diff
@@ -0,0 +1,97 @@
+from typing import List, LiteralString, Any
+
+from book import Book
+
+
+class Library:
+    """Class Library would LIST, LEND, RETURN or ADD book(s) to the library as per the input by the user"""
+
+    def __init__(self, listOfBooks: List[Book]):
+        self.availableBooks = listOfBooks
+        self.borrowed_books = []
+
+    def displayAvailableBooks(self):
+        for book in self.availableBooks:
+            if self.is_book_available(book):
+                print(f'Book: {book.name}')
+                print(f'Author: {book.author}')
+                print(f'Number of copies available : {book.quantity}')
+                print(f'Volume: {book.volume}')
+                print("--------------")
+
+    def lendBook(self, requestedBook):
+        for book in self.availableBooks:
+            if book.name == requestedBook:
+                if self.is_book_available(book):
+                    book.quantity -= 1
+                    if self.borrowed_books:
```

Comment:
Looks like Customer can be an object and it should be aware of its own borrowed books. Here borrowed_books seems to be a single object for the whole library.